api/src/iot/import_utils.py

```diff
@@ -604,14 +604,14 @@ def import_person(person_data: PersonData):
 
     names.append(person_data.last_name)
 
-    owner, _ = models.Person2.objects.update_or_create(
+    owner, _ = action_logger(models.Person2.objects.update_or_create(
```

Comment:
ik ben geen fan van deze setup omdat de `action_logger()` geen invloed heeft op de result. Het is wat dat betreft gek dat deze de result returned, maar ik snap je opzet wel. Met het oog het de einddatum van dit project vind ik het ok. Blijft het project langer live, dan moeten we dit meenemen in refactoring.

Reply:
Eens